katello-installer/katello-installer-base.spec

```diff
@@ -2,7 +2,7 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-#global prever .rc1
+# %%global prever .rc1
```

Comment:
@ehelms thoughts on this?  I like because I think its more obvious, and it only involves deleting characters rather than having to add characters.